ddtrace/contrib/flask/middleware.py

```diff
@@ -111,6 +111,7 @@ def _finish_span(self, response=None, exception=None):
             if span.sampled:
                 error = 0
                 code = response.status_code if response else None
+                method = request.method if response else None
```

Comment:
this will cause to have a `http.METHOD` tag == `'None'` in the UI. You may add `ok_(s.meta.get(http.METHOD) is None)` in a test where there is an exception to fix that.